asmcli/asmcli.sh

```diff
@@ -1084,15 +1090,21 @@ EOF
     warn "Missing one or more required options for [CLUSTER_LOCATION|CLUSTER_NAME|PROJECT_ID]"
   fi
 
-  if [[ -n "${KUBECONFIG}" ]]; then
-    KUBECONFIG_SUPPLIED=1
-  fi
-
   if [[ "${CLUSTER_DETAIL_SUPPLIED}" -eq 0 && "${KUBECONFIG_SUPPLIED}" -eq 0 ]]; then
     MISSING_ARGS=1
     warn "At least one of the following is required: 1) --kubeconfig or 2) --cluster_location, --cluster_name, --project_id"
   fi
 
+  if [[ "${KUBECONFIG_SUPPLIED}" -eq 1 && "${CONTEXT}" -eq 0 ]]; then
```

Comment:
I think this should be `-z "${CONTEXT}"` and not `"${CONTEXT}" -eq 0`.

Reply:
corrected!